cumulusci/core/runtime.py

```diff
@@ -38,30 +38,30 @@ def __init__(self, *args, load_keychain=True, **kwargs):
             self._load_keychain()
 
     @property
-    def universal_config_cls(self):
+    def universal_config_cls(self) -> Type:
         klass = self.get_universal_config_class()
         return klass or self.universal_config_class
 
     @abstractmethod
-    def get_universal_config_class(self):
+    def get_universal_config_class(self) -> Optional[Type]:
         return None
```

Comment:
I think `pass` or `...` works with `Type` here ([but not](https://github.com/python/mypy/blob/master/docs/source/class_basics.rst#abstract-base-classes-and-multiple-inheritance) with `super().get_universal...`) but no objection if we come back to it later